environment_linux.sh

```diff
@@ -84,6 +84,28 @@ check_circom_installed() {
     fi
 }
 
+check_rapidSnark_installed() {
+    echo "Check if rapidSnark is installed ..."
+    if [ ! -f "../packages/zkp/circom/unit_tests/rapidsnark/build/prover" ]; then
```

Comment:
I remember when we talked we said it need to be a seperate sh file, should not included in the environment_linux.sh?

And it has issue, environment_linux.sh is an environment setup. It does not rely on whether the lerna's setup.sh script had been run or not.

For this one, if we haven't do the setup.sh, it will failed as in fact when we setup the env, we should not have packages folder.

However,we should be able to install it.
I suggest we create a seperate folder tools/rapdisnark under lerna directory and install it. (Then need change the prove rapidsnark shell to use lerna's rapidsnark).

Reply:
Next time, for all the env shell developing, please test it in the following env at least:
1. Fresh clean lerna env.
2. Env installed and setup lerna env and re-run the env script

Please make sure no error happen in these 2 scenaro.